projects/dxilconv/lib/DxilConvPasses/DxilCleanup.cpp

```diff
@@ -1307,6 +1309,8 @@ bool DxilCleanup::IsDxilBitcast(Value *pValue) {
       case OP::OpCode::BitcastI32toF32:
       case OP::OpCode::BitcastI64toF64:
         return true;
+      default:
+        break;
```

Comment:
Nit: Would replacing this break; with return false; improve efficiency?

Reply:
I would trust the compiler to do the right thing and hesitate to make that change since we could theoretically add additional code afterward that we might want to run in this case. 

However, since this is a case that we don't expect to happen, an assert or unreachable might be appropriate as it might help us catch a bug if ever we add a bitcast

Reply:
Changed to return false to improve code reading efficiency.